src/style/global-style.js

```diff
@@ -34,6 +34,33 @@ button {
   50% { opacity: 0.6 }
   to { opacity: 0.85 }
 }
+
+a {
+  border-left: 1px solid #dddddd;
+}
+
+*:focus {
+  /* Provide a fallback style for browsers
+    that don't support :focus-visible */
+  outline: 2px solid red;
+  background: transparent;
+}
+
+@supports selector(*:focus-visible) {
+  *:focus, button:focus, select:focus, input:focus, a:focus, svg:focus  {
+    /* Remove the focus indicator on mouse-focus for browsers
+       that do support :focus-visible */
+    outline: none !important;
+    background: transparent;
```

Comment:
Why `background: transparent`? 

I also observed that a red button, when focused, appears green:

![image](https://user-images.githubusercontent.com/7419664/162686669-6048ef51-4f66-4217-9fcb-50226952c795.png)

Could this be related?